model/roots.js

```diff
@@ -51,9 +51,31 @@ async function getVerseRootWords(data) {
         })
     );
 }
+async function getRootsMeaning(data){
+    const sentences = [];
+    var all_roots = getVerseRootWords(data);
+    for (let root of all_roots){
+        let word = root.word                //get the word of the root (idk how to access the variable)
+        let rootmeaning = root.description  //get the meaning of the root  (idk how to access the variable)
+        let sentence  = "The word" + word + "comes from the root" + root + "and is associated with the meanings of:" + rootmeaning
+        sentences.push(sentence)
+    } 
+    var params = [data.verse_id];
```

Comment:
So, since we've already called `getVerseRootWords`, we don't really need to set any parameters, the SQL was already set and executed in that function call.